packages/game/src/common/gamewrapper.ts

```diff
@@ -49,7 +49,9 @@ function getTime({ G, ctx, playerID, events }: any) {
   G.millisecondsRemaining = new Date(G.end).getTime() - new Date().getTime();
 }
 
-export function gameWrapper<T_SpecificGameState>(game: GameType<T_SpecificGameState>): Game<T_SpecificGameState & GameStateMixin> {
+export function gameWrapper<T_SpecificGameState>(game: GameType<T_SpecificGameState>, 
+                                                 sendStrategyFunction = (...inputs: any[]): void => {},
```

Comment:
This typing looks weird, but I do not have any better idea yet. Just leaving this comment here, maybe I will have a better suggestion.